### PR TITLE
Fix "Add to scene" option while dropping model

### PR DESCRIPTION
**Pull request by Atlante45**
Fixes a bug where the model wouldn't actually be dropped in the scene.

### CODE REVIEW

**Comment by Atlante45** [comment]
https://app.asana.com/0/26551373415487/86331354879296

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -4333,8 +4333,8 @@ void Application::modelUploadFinished(AssetUpload* upload, const QString& hash)
     auto filename = QFileInfo(upload->getFilename()).fileName();
 
     if ((upload->getError() == AssetUpload::NoError) &&
-        (upload->getExtension().endsWith(FBX_EXTENSION, Qt::CaseInsensitive) ||
-         upload->getExtension().endsWith(OBJ_EXTENSION, Qt::CaseInsensitive))) {
```
Comment (Atlante45) [inline interface/src/Application.cpp]: Constants have dots included but not `getExtension()` so I simply switched them.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 913172b4908e3d67b57e2b7c25e5ab212c8be382 available at: https://gist.github.com/83c7cb1083f9a445ee59<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7075.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7075.dmg

File: interface/src/Application.cpp [hunk 2]
```diff
@@ -4333,8 +4333,8 @@ void Application::modelUploadFinished(AssetUpload* upload, const QString& hash)
...
+        (FBX_EXTENSION.endsWith(upload->getExtension(), Qt::CaseInsensitive) ||
+         OBJ_EXTENSION.endsWith(upload->getExtension(), Qt::CaseInsensitive))) {
```
Comment (huffman) [inline interface/src/Application.cpp]: This reads like a yoda conditional

**Comment by OmegaHeron** [comment]
Tested windows 7 against production build 4172.  Dragging/dropping a model to this PR works - model is placed in scene (if option to do so is checked).

File: interface/src/Application.cpp (hunk 2 above)
Comment (Atlante45) [inline interface/src/Application.cpp]: Indeed, like a yoda conditional it reads.

**Pull request merged**